CSSintroAss.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <title>DeliciousBlog</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="style.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <img src="images/logo.png">
+        <nav>
+            <img src="images/fb-icon.png"/>
+            <img src="images/twit-icon.png"/>
+            <img src="images/gp-icon.png"/>
+            <img src="images/insta-icon.png"/>
+            <img src="images/flic-icon.png"/>
+            <img src="images/pint-icon.png"/>
+            <img src="images/rss-icon.png"/>
+            <img src="images/mail-icon.png"/>
+        </nav>
+        <h1 id "blogName">Delicious</h1>
+        <h1 id "blogDesc">THE BEST FOOD BLOG ON THE WEB</h1>
+        <h2 id "recipeFor">Chocolate Pizza</h2>
+        <h2 id "postDate">Posted on Dec 15 2013/Desserts</h2>
+            <img src="images/print-icon.png">
+            <img src="images/choco-pizza.png">
+            <p id "stepOne" class "content">For the fig-swirl: melt butter over medium heat in a saucepan. Add brownsugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequenty, until you have a chuncky, jammy-mixture. Add salt with one or stirs,  set aside and let cool completely.</p>
+            <p id "stepTwo" class "conetent">Icecream: In a small pot over medium heat, combine milk, and granulated sugar until the sugar in completely dissolved and the milk is just barely luke-warm. Whisk in the egg yokes.Set mixture in the fridge and wait until the fig mixture is cooled.</p>
+            <p id "combSteps" class "content">Using an icecream machine, pour liquids into the frozen basinand proccess according to manufacturer instructions, i.e. let spin and thicken for 20 minutes before adding, mascapone, fig mixture and the nuts. Continue to proccess for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
+            <div class = "container"><img src="images/list-bg.png"/>
+                <ul id = "groceries">
+                    <li input type ="checkbox" id "itemOne"    class ""> 1 1/2 cups milk</li>
```

Comment:
Attribute syntax in html:
```
id="value"
```
All your id's are missing the "=" sign. Also, not sure why they all have id's.